src/vue-shims.d.ts

```diff
@@ -3,12 +3,9 @@
  * SPDX-License-Identifier: MIT
  */
 
-declare module '*.vue' {
-	import Vue from 'vue'
-	export default Vue
-}
+declare module '@nextcloud/vue/components/NcPasswordField' {
+	import type { Component } from 'vue'
+	const component: Component
```

Comment:
Why is this needed? Isn't it exposed from `@nc/vue` already?

Reply:
That component is not yet migrated :pensive: 
https://github.com/nextcloud-libraries/nextcloud-vue/issues/841